c++/src/kj/async-test.c++

```diff
@@ -884,40 +886,68 @@ TEST(Async, TaskSet) {
 // These tests require either fibers or pthreads in order to limit the stack size. Currently we
 // don't have a version that works on Windows without fibers, so skip the tests there.
 
-template <typename Func>
+inline size_t getSmallStackSize() {
+#if !_WIN32
+  // pthread_attr_setstacksize() requires a stack size of at least PTHREAD_STACK_MIN, which can
+  // vary by platform.  We'll clamp that to a reasonable range for stack overflow tests, and skip
+  // the pthread-based tests if we can't get it.
+  return kj::max(16 * 1024, kj::min(256 * 1024, PTHREAD_STACK_MIN));
```

Comment:
Using `PTHREAD_STACK_MIN` results in a compiler failure on NetBSD.

Also see:
- https://gnats.netbsd.org/37169
- https://mail-index.netbsd.org/tech-userlevel/2014/08/14/msg008726.html